### DISCUSSION TITLE
taking annotation during TTS is not possible, text highlighting grabbed by the tts engine

### DISCUSSION THREAD

**Discussion by panaC** [edrlab/thorium-reader#2691] [Developer corner]
<img width="1092" alt="image" src="https://github.com/user-attachments/assets/ef1714f8-2c5e-4a98-a6e4-ecde77d5dc7a">

**Comment by danielweck** [comment]
Correct, same with hyperlinks click. Same "problem" (feature?) with EPUB3 Media Overlays playback.  Moving this issue to a discussion as we need to figure out the interaction design (the click functionality is mutually-exclusive depending on the "mode" the user is in, for example click-to-play currently takes precedence, playback must be stopped for other baseline features to work)
